_maps/RandomRuins/IceRuins/skyrat/icemoon_underground_syndicate_base1_skyrat.dmm

```diff
@@ -1978,6 +2016,12 @@
 	dir = 1
 	},
 /obj/machinery/light/directional/south,
+/obj/structure/table,
+/obj/machinery/fax{
+	fax_name = "Interdyne";
```

Comment:
interdyne always calls themselves 'Interdyne Pharmaceuticals" when talking to non-syndies, so the fax should read the same